src/components/rich-radio/RichRadio.tsx

```diff
@@ -63,14 +63,14 @@ export const RichRadio = <T extends string>({
     >
       {items.map(item => (
         <button
-          key={item.value}
           className={clsx(
             styles.styledItem,
             itemHeight === 40 && styles.smallItem,
           )}
           data-disabled={item.disabled}
           data-state={item.value === selectedValue ? 'checked' : ''}
           data-tip={item.tooltip}
+          key={item.value}
```

Comment:
I think we want key to be in front of everything, it looks like the eslint rule is not working right

Reply:
![image](https://github.com/user-attachments/assets/f5eae569-9240-42e5-9707-fa8799085456)